### PR TITLE
Build the edgex page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Build the EdgeX page
**Drive by** Fix the WSL base template

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/edgex
- See that the content matches the [copydoc](https://docs.google.com/document/d/17tvfJBC5VncEVWkEsbyW_slAFFSoWV3jQUN7sa8nkOs/edit#heading=h.vt8za0d0luyb)

## TODOs
- Add the newsletter sign up form
- Update the icons

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8208

### CODE REVIEW

**Comment by cristinadresch** [comment]
@meralonso also for the sign up button the plan is to add a strip at the bottom with the sign up header and field, you think that's ok? so that button would anchor users down

**Comment by meralonso** [comment]
> @meralonso also for the sign up button the plan is to add a strip at the bottom with the sign up header and field, you think that's ok? so that button would anchor users down

@anthonydillon @cristinadresch what is the sign up header and field here when you press sign up button?

**Comment by anthonydillon** [comment]
@meralonso @cristinadresch I have added the sign up form to the page. Let me know what you think and how it can be improved.

**Comment by cristinadresch** [comment]
@anthonydillon i think this is good! do you need still any info from marketo to complete the form? not sure what the submitted state should be, but at the moment it isnt actually submitting right

**Comment by meralonso** [comment]
@anthonydillon can the scroll down be a bit smoother? Also, I'd change a few things on the layout, does it need to be designed and coded by tomorrow?

**Comment by meralonso** [comment]
> @meralonso I have applied your design 👍

Thanks @anthonydillon that was fast! It's looking good - just a couple of things:

 - Could you align these two modules? One seems to be higher than the other one
Screenshot 2020-09-03 at 17 26 06

- Can you put back the suru shape at the very bottom? I think it looked nice and I forgot to mention it

**Comment by anthonydillon** [comment]
@meralonso I have updated them to be all the same height but the "Built like cloud-native" icon has some white space around it.

**Comment by meralonso** [comment]
> @meralonso I have updated them to be all the same height but the "Built like cloud-native" icon has some white space around it.

@anthonydillon great, 120px high is a good height for images. Built like cloud-native icon [here](https://assets.ubuntu.com/v1/7370594b-2+-+Build+your+own+stack+and+prototype+your+application.svg)

I think you missed my one of my previous comments - Can you put back the suru shape for the CTA module at the very bottom?

**Comment by anthonydillon** [comment]
@meralonso thanks, ah of course. All done.

**Comment by meralonso** [comment]
> @meralonso thanks, ah of course. All done.

@anthonydillon thanks, all good from my side

**Comment by solazio** [comment]
The `contact us` and `subscribe now` buttons do not work

**Comment by anthonydillon** [comment]
Thanks @solazio good spots. Addressed the issues.

File: templates/internet-of-things/edgex.html [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+
+  <div class="row">
+    <div class="col-12 u-align--center">
+      <a href="#subscribe">Sign up for latest news&nbsp;&rsaquo;</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
```
Comment (solazio) [inline templates/internet-of-things/edgex.html:183]: Is there any reason why you don't use `u-fixed-width` instead in these cases?

**Review by solazio — APPROVED** [review]
Looks good :+1:. Thanks

File: templates/internet-of-things/edgex.html (hunk 1 above)
Comment (anthonydillon) [reply templates/internet-of-things/edgex.html:183]: Just personal preference. No difference really.

**Pull request merged**